lib/common/client/middleware/response/mhv_xml_html_errors.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+module Common
+  module Client
+    module Middleware
+      module Response
+        class MhvXmlHtmlErrors < Faraday::Response::Middleware
+          ERROR_LIST = %w(service_outage generic_xml generic).freeze
+
+          def on_complete(env)
+            return if ok_or_json?(env)
```

Comment:
I'd rather it got tripped up here, since really this should be the middleware to convert all non-json responses, should that ever happen. But that's fine, I can check for the presence of either HTML or XML here and pass everything else along.